### ISSUE TITLE
individual resource config is not specifiable in the master Manifest of deployment spec

### ISSUE THREAD

**Issue by cookbook87** [litmuschaos/litmus#3166]
**What happened**:
in some env its required to specify resources for cluster policies. currently deployment decides internally the flexibility is not given to specify the resource in manifest. this is why the replicas are not created for 

chaos-exporter-8c64544fc                    
chaos-operator-ce-7747546965                 
event-tracker-6467d9cdcd                     
subscriber-5df687bc48                       
workflow-controller-85c49cd57f               

**What you expected to happen**:
this needs to be corrected.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Comment by ksatchit** [comment]
Here are some resource recommendations you could use @cookbook87 . It is not specified in the master manifest to allow for installation in varied env (including min. resourced clusters on docker desktop/minikube etc.,) 

https://docs.litmuschaos.io/docs/user-guides/chaoscenter-cluster-scope-installation#resource-requirements-of-control-plane-components

https://docs.litmuschaos.io/docs/user-guides/chaosagents-installation#resource-requiremenets-for-agents-plane-components

**Comment by cookbook87** [comment]
Yeah but in a secured environment there always be policies which will obstruct to create resources internally unless they are not specified in the spec. 

Nevertheless, I am not even able to edit the deployment with required resources on the go.
